src/Cake.Common.Tests/Unit/Tools/OctopusDeploy/OctoCreateReleaseTests.cs

```diff
@@ -189,7 +190,7 @@ public void Should_Add_Username_To_Arguments_If_Not_Null()
                 // Then
                 Assert.Equal("create-release --project \"testProject\" " +
                              "--server http://octopus --apiKey API-12345 " +
-                             "--username \"mike123\"", result.Args);
+                             "--user \"mike123\"", result.Args);
```

Comment:
According to [octo.exe documentation](http://docs.octopusdeploy.com/display/OD/Deploying+releases) it should have been `--user` all along:
![2016-09-28 23_40_43-deploying releases - octopus deploy 3 0 - octopus deploy](https://cloud.githubusercontent.com/assets/3606356/18935009/148a05f0-85d5-11e6-9124-2c8a868f7b1d.png)